client/src/components/CourseContent.jsx

```diff
@@ -18,8 +18,8 @@ class CourseContent extends React.Component {
       course: {},
       isLoaded: false,
       allExpanded: false,
-      host: 'ec2-18-130-234-175.eu-west-2.compute.amazonaws.com:9800',
-      // host: '127.0.0.1:9800',
+      // host: 'ec2-18-130-234-175.eu-west-2.compute.amazonaws.com:9800',
+      host: '127.0.0.1:9800',
```

Comment:
not sure I understand why you had to switch back to localhost. Is that temporary while working on it locally?

Reply:
Whoops didn't mean to do this